lib/msal-browser/docs/v4-migration.md

```diff
@@ -8,7 +8,11 @@ If you are coming from MSAL v4, you can follow this guide to update your code to
 
 ## API Breaking Changes
 
-[TBD]
+### TokenCache and loadExternalTokens
+
+The `TokenCache` object as well as the `getTokenCache` function in `PublicClientApplication` have been removed from MSAL.js.
```

Comment:
```suggestion
MSAL JS API for [loadExternalTokens](../testing.md#the-loadexternaltokens-api) is modified. The changes include:
* Removed `TokenCache` object and `getTokenCache()` support from PCA
* `loadExternalTokens()` accepts `Configuration` (add reference to code) as a parameter instead of `TokenCache`
```

And +1 to Thomas's comment below, add a before/after code fragment to highlight the new usage.

Reply:
Done!